README.md

```diff
@@ -23,8 +23,7 @@ module "folders" {
   source  = "terraform-google-modules/folders/google"
   version = "~> 0.1"
 
-  parent_id   = "65552901371"
-  parent_type = "folder"
+  parent_id  = "65552901371"
```

Comment:
Wouldn't this need to be:

```suggestion
  parent  = "folders/65552901371"
```

Reply:
Done, usage updated and pinned to the new release 2.0